crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs

```diff
@@ -249,14 +241,7 @@ impl<T: TransportConnect> SequencerAppender<T> {
                     continue;
                 }
                 StoreTaskStatus::Sealed(_) => {
-                    tracing::trace!(node_id=%server.node_id(), "node is sealed");
-                    continue;
-                }
-                StoreTaskStatus::AdvancedLocalTail(_) => {
-                    // node local tail is behind the batch first offset
-                    // question(azmy): we assume node has been replicated?
-                    checker.set_attribute(node_id, true);
-                    pending_servers.remove(&node_id);
+                    tracing::trace!(node_id=%server.node_id(), "store task cancelled duo to sealing");
```

Comment:
Once a loglet gets sealed, are we gonna stop this task somehow or is this part missing?

Reply:
After discussing this with @AhmedSoliman we agreed that the appended will keep retrying forever (as long as the task is alive) in case not all log servers are sealed and/or dead. 

It's only cancelled when the parent task is cancelled by a higher power.

I will make sure the appended task is spawned as a child to get cancelled with the parent task.

> I am still not sure about the behaviour of the child task if it's automatically dropped when the parent is cancelled, or the child still need to check a cancellation token.

Reply:
I think that the child task needs to select over the cancellation token/react to it explicitly.

Reply:
Yes, I already verified this with @AhmedSoliman, I will add the code to handle the child shutdown on cancellation

Reply:
@tillrohrmann I updated the code to handle parent cancellation. Please let me know if you still have more comments. And thank you so much for your review :)

Reply:
For future reference and to elaborate on what I want to happen when sealing is taking place, once we learn that a seal is happening:
1- Reject new appends
2- In-flight SequencerAppenders should continue to store the record unless it becomes impossible (f-majority of sealed Stored responses)
3- Aid in sealing (run a SealTask) if it detects that _some_ nodes are sealed (optional/optimization)